distribution/docs/src/main/resources/content/_coreConcepts/security-standards.adoc

```diff
@@ -0,0 +1,47 @@
+:type: subCoreConcept
+:status: published
+:title: Security Standards
+:parent: Standards Supported by ${branding}
+:order: 03
+
+== {title}
+
+${branding} makes use of these security standards to protect the system and interactions with it.
+
+
+.Secure Protocols
```

Comment:
I had the radical idea of punting on organizing these standards into categories by functionality. Instead, organize them by standards body, like this:

Standard           | Standards Body | Version(s) Supported
----------------|----------|-----------------------
SAML                 | OASIS     | 2.0
WS-Policy         | OASIS      | ...
IETF                   | ...             | ...

Reply:
I'm not opposed.